project/Build.scala

```diff
@@ -193,15 +193,20 @@ object Build {
       "-feature",
       "-deprecation",
       "-unchecked",
-      //"-Wconf:cat=deprecation&msg=Unsafe:s",    // example usage
-      "-Werror",
+      //"-Werror",
       //"-Wunused:all",
-      //"-rewrite", // requires -Werror:false since no rewrites are applied with errors
       "-encoding", "UTF8",
       "-language:implicitConversions",
+      "--java-output-version", Versions.minimumJVMVersion,
```

Comment:
Not very important, but typically I see usage `key:value` settings more readable when analysing `show scalacOptions` in sbt since these are showed as single string, when compared to 2, implicitlly concanated settings, that span 2 lines when inspecting. 

```suggestion
      s"--java-output-version:${Versions.minimumJVMVersion}",
```